typescript/agentkit/src/action-providers/flaunch/README.md

```diff
@@ -0,0 +1,109 @@
+# Flaunch Action Provider
+
+This directory contains the **FlaunchActionProvider** implementation, which provides actions for interacting with the Flaunch protocol on Base network. Flaunch is a protocol for launching and trading memecoin tokens.
+
```

Comment:
Please add the env dependency info here, including a link to the pinata website,

Add the section `### Environment Variables` underneath the `## Overview` section

e.g. https://github.com/coinbase/agentkit/blob/main/typescript/agentkit/src/action-providers/onramp/README.md#environment-variables